inference-engine/src/transformations/src/transformations/gru_cell_decomposition.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/gru_cell_decomposition.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <transformations/utils/utils.hpp>
+
+ngraph::pass::GRUCellDecomposition::GRUCellDecomposition() {
+    auto gru_cell = ngraph::pattern::wrap_type<opset4::GRUCell>();
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
```

Comment:
As you don't capture anything
```suggestion
    ngraph::matcher_pass_callback callback = [](ngraph::pattern::Matcher& m) {
```